src/app/projects/pages/projects-list/projects-list.component.html

```diff
@@ -104,7 +104,8 @@
         </td>
 
         <td class="vf-table__cell">
-          <p>{{ project.cellCount ? (project.cellCount | number) : "-" }}</p>
+          <span *ngIf="project.cellCount">{{ (project.cellCount | number) }}</span>
```

Comment:
```suggestion
         <span *ngIf="project.cellCount; else noCellCount">{{ (project.cellCount | number) }}</span>
         <ng-template #noCellCount>-</ng-template>
```
and remove the next line